packages/css/radiobutton.css

```diff
@@ -25,12 +31,12 @@
     height: 1rem;
     width: 1rem;
 
-    border: 0.125rem solid;
```

Comment:
Radioknappene vises ikke lenger riktig i firefox. På master vises dotten for en avmerket radioknapp på riktig sted, mens på denne branchen så vises den litt forskjøvet. Jeg vil tro det er pga denne, eller en av de påfølgende endringene som endrer litt på størrelser og verdier.

Reply:
Ser fremdeles feil ut hos meg dessverre :( 
![bilde](https://user-images.githubusercontent.com/32133041/221509104-625cab21-5e5d-41ae-a0a8-7c6fa68e226b.png)

Reply:
I chrome ser det riktig ut, men i Firefox er det som på bildet over, så det er tydeligvis en eller annen forskjell mellom browserene.